components/ws-manager-bridge/src/container-module.ts

```diff
@@ -91,4 +92,6 @@ export const containerModule = new ContainerModule((bind) => {
     bind(DebugApp).toSelf().inSingletonScope();
 
     bind(Client).toDynamicValue(getExperimentsClientForBackend).inSingletonScope();
+
+    bind(WorkspaceInstanceController).to(WorkspaceInstanceControllerImpl).inTransientScope();
```

Comment:
What does transient mean?

Reply:
A new instance is created for every request. Will explain a bit in the documentation.